Makefile

```diff
@@ -19,5 +19,19 @@ rest/swagger.json:
 	./scripts/patch-skf-uuids.sh "$@"
 	./scripts/patch-x-nullable.sh "$@"
 
+v2/rest/models/: v2/rest/swagger.json
+	$(RM) -rf "$@" && $(MKDIR) -p "$@"
+	$(DOCKER) run --rm \
+		--volume "$(shell pwd):/src" \
+		--user "$(shell id -u):$(shell id -g)" \
+		quay.io/goswagger/swagger:v0.25.0 \
+			generate model --spec="/src/$<" --target="/src/$@.."
+
+v2/rest/swagger.json:
+	$(WGET) "$(API_URL)/v2/docs/swagger/doc.json" -O "$@"
+	./scripts/patch-skf-uuids.sh "$@"
+	./scripts/patch-x-nullable.sh "$@"
+
```

Comment:
Should these live in its own Makefile in the v2 folder?

Then you can redirect the call by having a v2: step in the outer makefile.

Reply:
maybe felt like it just make it more complicated with more make files.
Maybe v2 should be in another repo?

Reply:
hmm. I noticed that the rule for v1 and v2 is the same? You can just merge them into one rule if you want,
```Makefile
rest/models v2/rest/models: %/../swagger.json
	$(RM) -rf "$@" && $(MKDIR) -p "$@"
	$(DOCKER) run --rm \
		--volume "$(shell pwd):/src" \
		--user "$(shell id -u):$(shell id -g)" \
		quay.io/goswagger/swagger:v0.25.0 \
			generate model --spec="/src/$<" --target="/src/$@.."
```

Reply:
learn something everyday, will do.